### ISSUE TITLE
Build fails on restructure branch

### ISSUE THREAD

**Issue by nuernbergk** [kztk-m/flippre#6]
The command `stack build` always fails on the restructure branch for me:

```
% stack build               
No information from Hackage index, updating
Selected mirror https://hackage.haskell.org/
Downloading timestamp
Waiting to acquire cache lock on /Users/nn/.stack/pantry/hackage/hackage-security-lock
Acquired cache lock on /Users/nn/.stack/pantry/hackage/hackage-security-lock
Released cache lock on /Users/nn/.stack/pantry/hackage/hackage-security-lock
No package index update available, but didn't update cache last time, running now
Calculating hashes to check for hackage-security rebases or filesystem changes
No old cache found, populating cache from scratch
Populating cache from file size 578792320, hash 2641710d953dee046b5d01a6b57a679ada230db8efe26e29c7ae7ede59ab0348
Populating package index cache ...
Failed populating package index cache
IncompletePayload 578793344 818
```

This happens even on a clean clone of the repository (no cache files etc).
The master branch builds as expected.

**Comment by kztk-m** [comment]
Thank you for reporting. I have not tested Stack well; in fact, I haven't added external package information (regarding https://github.com/kztk-m/EbU) to `stack.yaml` yet. Currently, I am developing the package with `cabal`, and thus, `cabal build` should work instead. 

By the way, since the error message does not mention this package, it seems to me that the error comes from Stack itself. Googling the error message "Failed populating package index cache", I found https://www.reddit.com/r/haskellquestions/comments/ed71ok/unable_to_add_classyprelude_dependency/, which suggests removing "~/.stack". But, I am not sure this workaround works for this case. 

Anyway, since `stack.yaml` has not been configured well for now, as written above, even after this issue has been resolved, I don't think `stack build` works.
